src/bci_build/package.py

```diff
@@ -2016,6 +2048,50 @@ def _get_os_container_package_names(os_version: OsVersion) -> tuple[str, ...]:
     for os_version in ALL_BASE_OS_VERSIONS
 ]
 
+# https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3991.pdf
+# Chapter 9.1 Crypto Officer Guidance
+_FIPS_15_SP2_ASSET_BASEURL = "https://api.opensuse.org/public/build/"
```

Comment:
Hm, downloading binaries from build.o.o that way isn't great...

Using an aggregate to let OBS handle this isn't great either though.

Reply:
it works great.